kapitan/utils.py

```diff
@@ -267,7 +281,7 @@ def deep_get(dictionary, keys, previousKey=None):
     return value
 
 
-def searchvar(flat_var, inventory_path, pretty_print):
+def searchvars(flat_var, inventory_path, pretty_print):
```

Comment:
Why rename?

Reply:
In order to use fire I have to create a wrapper around the original imported searchvar. I will use an alias for it while importing to remove name conflict.